proxy/proxy.go

```diff
@@ -34,6 +40,40 @@ func formatRequest(r *http.Request) string {
 	return strings.Join(request, "\n")
 }
 
+func pollConfigChange(config *Config) {
+	//we check for file changes every half seccond
+	ticker := time.NewTicker(500 * time.Millisecond)
```

Comment:
I would turn `500` into a Constant with a descriptive name then we can avoid the comment above

Reply:
👍